Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM        debian:bookworm
+
+VOLUME      ["/server"]
+
+WORKDIR     /usr/src/SpadesX
+
+RUN         set -ex; \
+            apt -y update; \
+            apt -y upgrade; \
+            apt install build-essential make cmake zlib1g-dev libjson-c-dev libreadline-dev libbsd-dev git; \
+            git clone https://github.com/SpadesX/SpadesX.git; \
```

Comment:
I wouldn't recommend doing a `git clone`, as it requires installing an extra package, and doesn't let you test your local changes. Instead, you should do `COPY . /usr/src/spadesx`.

Reply:
far better, and quicker, thanks.